### PR TITLE
Update docs to match impl, fixes for React 16.

### PR DESCRIPTION
**Pull request by AaronFriel**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (neoziro) [inline .vscode/settings.json]: Can you remove this file?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### 3.0.0-rc.0
+
+* The React component `AppContainer` is now `HotContainer`.
+* The exported Babel plugin is now `react-hot-loader/babel-plugin`.
```
Comment (neoziro) [inline CHANGELOG.md]: Why renaming "babel" to "babel-plugin"? I think it doesn't have so much value except breaking all existing installation of react-hot-loader.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 ## Changelog
 
+### 3.0.0-rc.0
+
+* The React component `AppContainer` is now `HotContainer`.
```
Comment (neoziro) [inline CHANGELOG.md]: We have to expose both AppContainer and HotContainer and deprecate AppContainer. The 3.0.0 beta is widely used and we should minimize breaking changes and take users by the hand.

Comment (AaronFriel) [reply CHANGELOG.md]: https://github.com/gaearon/react-hot-loader/pull/639

You're literally the person who deleted AppContainer?

File: CHANGELOG.md (hunk 2 above)
Comment (AaronFriel) [reply CHANGELOG.md]: https://github.com/gaearon/react-hot-loader/pull/654/commits/dc1daaa61bddf51a2f1fa0a35d533ceadba44055

**Comment by AaronFriel** [comment]
You're jerks, I'm out.

**Pull request closed without merging**